lola.html

```diff
@@ -0,0 +1,953 @@
+        </li>
+        <li>
+          Aurora’s Activity could move, and Cherry's server could learn about the move and know that it has Like activities referencing the original article, and nevertheless <b>leave them alone</b>.  They already propagated, their purpose is served.
+        </li>
+      </ol>
+
+      <p>
+        Existing servers seem to follow the “<b>leave them alone</b>” model rather than the <b>“rewrite”</b> model.  The way items propagate initially doesn’t have to define the way they are maintained.  In any case, trying to follow the rewrite model would sure involve some misses, failing back to the leave them alone model.
+      </p>
+
+      <p>
+        For ease of implementation, we lean towards assuming the “leave them alone” model.  Some of the reasons why:
+      </p>
+
+      <ul>
+        <li>How far should change propagation go? If Aurora moves her article and Cherry's Like gets updated does that propagate to Aurora’s subscribers? Do they get notified? </li>
+        <li>Where should change propagation go to?  Original follower list at the time the item was sent out? Aurora’s follower list today? 
+        </li>
+        <li>A rewrite model, if it were to be relied upon, would require many OTHER servers to participate in a data transfer specification, not just the two servers directly involved.  This increases the difficulty of adoption.
+        </li>
+        <li>
+          Since Activities/Likes can be disconnected in other ways, supporting breadcrumbs seems more useful than trying to achieve perfect rewriting.
+        </li>
+      </ul>
+
+      <p>
+        In addition to assuming that 3rd party servers leave their likes alone in many cases, this specification recommends that the destination server add breadcrumbs to Activity and Like objects to record their prior identities. This is beneficial in showing a nuanced view of older data.  If necessary, a server interested in validating a collection of “Likes” submitted to it, can mostly do so (except for accounts that have disappeared) if the object that the collection of “Likes” it is on maintains a history of past object IDs.  Note that this still ALLOWS other servers to fix up their Likes but they can do so immediately, later, or never.
+      </p>
+
+      <h3>Data Degradation</h3>
+        <p>Any account migration bears a risk of data loss or degradation.  A server cannot reasonably accept content it does not know how to store or present.  A roundtrip can result in data that is not in all ways, or even in important ways, identical to the data before it was migrated and migrated back.  This specification is intended to facilitate best-effort account migration rather than perfect duplication, with the philosophy that the perfect can be the enemy of the good.</p>
```

Comment:
Here I kind of disagree-- I would argue that the server can ACCEPT the content, and maybe even set it aside in some kind of "DO NOT OPEN, MAY CONTAIN SNAKES" bucket, it just can't reasonably INGEST or USE that content.

The line gets a little hard to draw when you consider an Activity or Object might be mostly intelligible, but have one extra property defined in a FEP the destination server didn't implement. Here, maybe a second bucket is needed, labeled "LOST IN TRANSLATION", where the complete form of the Activity or Object might get put, if that unknown property were dropped before ingesting the intelligible rest of it.

This is the kind of corner-case hell I opened #14 to torment you with 😊

Reply:
(but sure, maybe the spec can mention these two buckets as MAY options, and not require them? more important, perhaps, is surfacing to the user a notification that properties have been dropped from 110 of the 2135 imported objects, or that 255 additional objects will not be imported)